config/kubernetes-sigs/org.yaml

```
@@ -1006,6 +1006,7 @@ teams:
     - songjiaxun
     - sunnylovestiramisu
     - tyuchn
+    - sneha-at
```

PR checks failed because the list isn't sorted ([reference link](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/org/4799/pull-org-test-all/1763326149766680576)). Could you please make sure the members list is sorted after adding the new entry?

```suggestion
    - sneha-at
    - songjiaxun
    - sunnylovestiramisu
    - tyuchn
```